R/folders.R

```diff
@@ -0,0 +1,135 @@
+#' Functions to manipulate variables' folder structure
+#'
+#' Variables in Crunch datasets are organized into folders, like in a file
+#' system. These functions allow you to create new folders and move objects into
+#' folders. Their names, `mv` and `mkdir`, suggest their Unix file utility
+#' inspiration.
+#'
+#' The functions have some differences from the strict behavior of their Unix
+#' ancestors. For one, they work recursively, without additional arguments:
+#' `mkdir` will make every directory necessary to construct the requested path,
+#' even if all parent directories didn't already exist; and `mv` doesn't
+#' require that the directory to move to already exist. `mkdir` also takes an
+#' optional `variables` argument, which allows you to say "make this directory
+#' **and** put these variables in it" in one command. As a result of these
+#' behaviors, `mv` and `mkdir` here are essentially the same, just with
+#' arguments in reverse order: `mv` is move "variables" to "path", while `mkdir`
+#' is create "path" and put "variables" in it.
+#'
+#' Both functions take "dataset" as the first argument, so they naturally
+#' support pipelining (as with the \code{\%>\%} operator).
```

Comment:
Did `` `\%>\%` `` fail on this because of parsing?

Reply:
There was some failing so I went to magrittr source to see how they did their roxygen and stole it.